### PR TITLE
Docker dependencies

### PR DESCRIPTION
**Pull request by kokonakajima**
Adds more Docker dependencies for the `make` commands in `data/`, including a new dev-requirements.txt file with python packages used in the `data/make` scripts.

### CODE REVIEW

File: server/Dockerfile [hunk 1]
```diff
@@ -16,8 +16,16 @@ RUN apt update \
 
 # install data/make dependencies
 RUN npm install -g mapshaper
+RUN apt update && apt -y install jq
```
Comment (alisonag) [inline server/Dockerfile]: All of the apt installs can be added to the existing RUN apt command further up. [Here's a bit more info](https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better) on multiple vs single run commands and how they impact the docker build. Not sure how important it ultimately is in this scenario, but at the very least it lets you avoid running apt update twice

File: server/Dockerfile [hunk 2]
```diff
@@ -16,8 +16,16 @@ RUN apt update \
...
 COPY ./requirements.txt /usr/local/aclu/elections-api/requirements.txt
-RUN pip install -r requirements.txt
\ No newline at end of file
+RUN pip install -r requirements.txt
+COPY ./dev-requirements.txt /usr/local/aclu/elections-api/dev-requirements.txt
+RUN pip install -r dev-requirements.txt
+
+# smartcrop + dependencies
+RUN apt update && apt -y install libopencv-dev
+RUN npm install -g opencv --unsafe-perm=true
```
Comment (alisonag) [inline server/Dockerfile]: Similarly, you could merge the npm installs into one line (and possibly the pip installs too)

**Review by alisonag — COMMENTED** [review]
Once we are both back at the office (Wednesday) I'll need some help testing out the commands tied to these dependencies (maybe we could pair?). I also ran into some warning messages during the build I want to check with you about; I'm not sure if they are safe to ignore or not. But in the meantime I've left a few suggestions for consolidating commands in the dockerfile.

File: server/Dockerfile (hunk 1 above)
Comment (kokonakajima) [reply server/Dockerfile]: ah thanks yeah this makes sense! moved it up to the existing one

File: server/Dockerfile (hunk 2 above)
Comment (kokonakajima) [reply server/Dockerfile]: yeah totally, merged both!

**Comment by kokonakajima** [comment]
thanks for taking a look @alisonag!! i consolidated those commands you mentioned.

also leaving a note here from our conversation yesterday just for future reference: the warning messages have to do with opencv (maybe related to a node version??) but don't seem to affect the cropping script at least

**Review by alisonag — APPROVED** [review]
looks good!

**Pull request merged**